src/objective-c/tests/Podfile

```diff
@@ -91,13 +82,23 @@ post_install do |installer|
     target.build_configurations.each do |config|
       config.build_settings['GCC_TREAT_WARNINGS_AS_ERRORS'] = 'YES'
     end
-    if target.name == 'gRPC-Core' or target.name == 'gRPC-Core.default-Cronet-Interface-Cronet-Tests'
+    if target.name == 'gRPC-Core' or target.name == 'gRPC-Core.default-Cronet-Implementation-Cronet-Interface-Tests' or target.name == 'gRPC-Core.default-Cronet-Implementation'
```

Comment:
Including the comment will prevent people from going: "Huh? Do we have more than one pod that starts with gRPC-Core?". Plus it'll help debug things if Cocoapods changes that naming in the future, by explaining what our intention was.